lib/utils/ReporterStats.js

```diff
@@ -95,12 +95,19 @@ class ReporterStats extends RunnableStats {
             failures: 0
         }
         this.failures = []
+        this.passes = []
     }
 
     getCounts () {
         return this.counts
     }
 
+    getPasses () {
+        return this.passes.map(function (test) {
```

Comment:
Why `map()` with empty callback? Maybe just `return this.passes`?

Reply:
Fair point! Changed...